src/service/action-impl.js

```diff
@@ -49,6 +49,21 @@ const ELEMENTS_ACTIONS_MAP_ = {
   'AMP': ['setState'],
 };
 
+/** @const {!Object<string, !Object<string, string>>} */
+const WHITELISTED_INPUT_DATA_ = {
+  range: {
+    min: 'number',
+    max: 'number',
+    value: 'number',
+  },
+  radio: {
+    checked: 'boolean',
+  },
+  checkbox: {
+    checked: 'boolean',
+  },
```

Comment:
Nit: Use string literals for keys that should be constants. This future-proofs against Closure renaming.